thoth-export-server/src/specification_query.rs

```diff
@@ -55,7 +55,7 @@ impl SpecificationQuery {
         // get a vector of all page offsets we will need
```

Comment:
Is the +1 here (line 54) still needed? Where there are 20 works, the new logic produces offsets [0, 20] - shouldn't 20 works all fit into one page?

Reply:
l:54 gets you the number of pages. The logic to get the offsets simply gives you multiples of `PAGINATION_LIMIT`, so in your example, if work_count = 20, total_pages = 1 (20/100, floored +1), and offsets = [0]

Reply:
Ah, I was thinking of the UI's pagination limit, which is 20, not 100!

I meant that where `work_count` == `PAGINATION_LIMIT`, `offsets` comes out as `[0, PAGINATION_LIMIT]`, rather than as `[0]`. Is that right, or not? I no longer trust myself to unpick off-by-ones correctly...

https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=8cd989086fd1f6e0efd05a4cbf1e5971

Reply:
Yes, your right! I need to add an exeption for work_count == limit

Reply:
I think it's where the count is any multiple of the limit :thinking:

Reply:
Yes, `(work_count + PAGINATION_LIMIT - 1) / PAGINATION_LIMIT;` should do the trick